apps/files/src/components/FileList.vue

```diff
@@ -301,6 +308,32 @@ export default {
       }
 
       return `files-file-list-action-button-${id}`
+    },
+
+    $_resizeHeader () {
+      this.$nextTick(() => {
+        const headerRow = this.$refs.headerRow
+        const firstRow = this.$refs.firstRow
+        if (headerRow && firstRow) {
+          const headerColumns = Array.from(headerRow.$el.childNodes).filter(n => n.nodeType === Node.ELEMENT_NODE)
+          const rowColumns = Array.from(firstRow.$el.childNodes).filter(n => n.nodeType === Node.ELEMENT_NODE)
+          for (let i = 0; i < headerColumns.length; i++) {
+            const headerColumn = headerColumns[i]
+            const rowColumn = rowColumns[i]
+            if (!rowColumn) {
```

Comment:
Do we want to start syncing sizes, if the column layout doesn't match between header and body? Are there valid use cases for that? If not we could check this once before the for loop (break on `if (headerColumns.length !== bodyColumns.length)`) instead of checking in every iteration.